include/features.php

```diff
@@ -16,11 +16,11 @@
 function feature_enabled($uid, $feature) {
 	$x = Config::get('feature_lock', $feature);
 
-	if ($x === false) {
+	if (is_null($x)) {
 		$x = PConfig::get($uid, 'feature', $feature);
-		if ($x === false) {
+		if (is_null($x)) {
 			$x = Config::get('feature', $feature);
-			if ($x === false) {
+			if (is_null($x)) {
 				$x = get_feature_default($feature);
```

Comment:
While we're at it, why not rewrite this as
````php
$x = Config::get('feature_lock', $feature, PConfig::get($uid, 'feature', $feature, Config::get('feature', $feature,  get_feature_default($feature))));
````
😇

Reply:
This could be set on the list for later. This was the quickest and no so dirty clean up.

Reply:
This wasn’t serious! It would work, but it would be very inefficient and unreadable! 😅

Reply:
I really thought about it ;-) But I did a semi solution for the current PR.